src/index.html

```diff
@@ -12,7 +12,42 @@
       href="styles/index.scss"
     />
```

Comment:
You should link a CSS file in your HTML, not SCSS. The browser cannot interpret SCSS. Your SCSS files should be compiled into a CSS file which is then linked to your HTML.